src/utilities/getComputedStyleValue.ts

```diff
@@ -0,0 +1,22 @@
+// https://stackoverflow.com/a/62557418/3511012
+function globalExists(varName: string): boolean {
+  const globalEval = eval
+  try {
+    globalEval(varName)
+    return true
+  } catch {
+    return false
+  }
+}
```

Comment:
not a fan of this pattern, but this is carried over from our work in saving `crypto` from blowing up when running in node (like in tests). Note this is only used to ensure `window` actually exists and is not exported

Reply:
I could. I put it here to make it even harder to accidentally get exported and used somewhere that a user might provide the eval value.